tasks/facial_landmark_detection/model.py

```diff
@@ -0,0 +1,169 @@
+            dropout (float): The droupout probability
+
+        """
+        super().__init__()
+        # _log_api_usage_once(self)
+
+        if block is None:
+            block = CustomInvertedResidual
+
+        if norm_layer is None:
+            norm_layer = nn.BatchNorm2d
+
+        input_channel = 32
+        last_channel = 1280
+
+        if inverted_residual_setting is None:
+            inverted_residual_setting = [
+                # t, c, n, s, k
+                [1, 16, 1, 1, 3],
+                [6, 24, 2, 2, 3],
+                [6, 32, 3, 2, 3],
+                [6, 64, 4, 2, 3],
+                [6, 96, 3, 1, 3],
+                [6, 160, 3, 2, 3],
+                [6, 320, 1, 1, 3],
+            ]
+
+        # only check the first element, assuming user knows t,c,n,s,k are required
+        if len(inverted_residual_setting) == 0 or len(inverted_residual_setting[0]) != 5:
+            raise ValueError(
+                f"inverted_residual_setting should be non-empty or a 4-element list, got {inverted_residual_setting}"
+            )
+
+        # building first layer
+        input_channel = _make_divisible(input_channel * width_mult, round_nearest)
+        self.last_channel = _make_divisible(last_channel * max(1.0, width_mult), round_nearest)
+        features: List[nn.Module] = [
+            Conv2dNormActivation(3, input_channel, stride=2, norm_layer=norm_layer, activation_layer=nn.ReLU6)
+        ]
+        # building inverted residual blocks
+        for t, c, n, s, k in inverted_residual_setting:
+            output_channel = _make_divisible(c * width_mult, round_nearest)
+            for i in range(n):
+                stride = s if i == 0 else 1
+                features.append(block(input_channel, output_channel, stride, expand_ratio=t, kernel=k, norm_layer=norm_layer))
+                input_channel = output_channel
+        # building last several layers
+        features.append(
+            Conv2dNormActivation(
+                input_channel, self.last_channel, kernel_size=1, norm_layer=norm_layer, activation_layer=nn.ReLU6
+            )
+        )
+        # make it nn.Sequential
+        self.features = nn.Sequential(*features)
+
+        # building classifier
+        self.classifier = nn.Sequential(
+            nn.Dropout(p=dropout),
+            nn.Linear(self.last_channel, num_classes),
+        )
+
+        # weight initialization
+        for m in self.modules():
+            if isinstance(m, nn.Conv2d):
+                nn.init.kaiming_normal_(m.weight, mode="fan_out")
+                if m.bias is not None:
+                    nn.init.zeros_(m.bias)
+            elif isinstance(m, (nn.BatchNorm2d, nn.GroupNorm)):
+                nn.init.ones_(m.weight)
+                nn.init.zeros_(m.bias)
+            elif isinstance(m, nn.Linear):
+                nn.init.normal_(m.weight, 0, 0.01)
+                nn.init.zeros_(m.bias)
+
+    def _forward_impl(self, x: Tensor) -> Tensor:
```

Comment:
not really needed unless we plan to have subclasses of CustomMobileNetV2 right?

Reply:
@yrajas needs to take a look.

Reply:
Yes, this can be private method. The forward method is what gets overriden in the quantizable version. A new PR will be done for that.